content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,158 @@
-   If you are on Odoo Online and have the Enterprise version, you benefit from free credits to test our
-   IAP features.
+.. _in_app_purchase/portal:
 
-IAP accounts
+IAP services
 ============
 
-Credits to use IAP services are stored on IAP accounts, which are specific to each service.
-By default, IAP accounts are common to all companies, but can be restricted to specific
-ones. Activate the :ref:`developer mode <developer-mode>`, then go to :menuselection:`Technical
-Settings --> IAP Account`.
+The `Odoo IAP website <https://iap.odoo.com/iap/all-in-app-services>`_ lists all IAP services
+currently available, which include the following:
 
-.. image:: in_app_purchase/image2.png
-   :align: center
+- :guilabel:`AI generated call summary and transcript service`: call recording transcript service
+  for the *Asterisk Plus* application.
+- :guilabel:`Amazon Odoo Connector`: establishes seamless integration between an Amazon Seller
+  Central Account and Odoo, allowing users to effectively manage their products and orders.
+- :guilabel:`Documents Digitization`: digitizes scanned or PDF vendor bills, expenses, and resumes
+  with OCR and AI.
+- :guilabel:`Lead Generation`: generates leads based on a set of criteria and converts web visitors
+  into quality leads and opportunities.
+- :guilabel:`Partner Autocomplete`: automatically populates contact records with corporate data.
+- :guilabel:`SMS`: sends SMS text messages to contacts directly from the database.
+- :guilabel:`Signer identification with itsme®️`: adds an identification step in Odoo's *Sign* app
+  that asks signatories to provide their identity through the itsme®️ platform.
+- :guilabel:`Snailmail`: sends customer invoices and follow-up reports by post, worldwide.
+- :guilabel:`eInvoice Service`: submits invoices to Invoice Registration Portal (IRP).
 
-.. tip::
-   An IAP account can be disabled by appending `+disabled` to its token.
-   Reverting this change will re-enable the account.
+Use a service
+-------------
 
-IAP Portal
-==========
+IAP services are automatically integrated with Odoo and do not require users to configure any
+settings. To use a service, simply interact with it wherever it appears in the database.
 
-The IAP Portal is a platform regrouping your IAP Services. It is accessible from
-:menuselection:`Settings app --> Odoo IAP --> View my Services`. From there, you can view your current
-balance, recharge your credits and set a reminder when your balance falls below a threshold.
+.. example::
+   In this example, the SMS service is used from a contact's record.
 
-.. image:: in_app_purchase/image3.png
-   :align: center
+   .. image:: in_app_purchase/sms-icon.png
+      :align: center
+      :alt: Hover the mouse to the right of a contact's phone number to reveal the SMS phone icon.
+
+   1. Navigate to the *Contacts* application and click on a contact.
+   2. Hover the mouse over the phone number field and a :guilabel:`SMS 📱 (phone)`
+      icon appears.
```

Comment:
Might be overkill, but you may want to mention this only appears if there is a phone number populating the field. if it's blank this doesn't appear.